src/utils/path.utils.ts

```diff
@@ -56,6 +56,11 @@ export function getScopeAndLangFromPath({
 }
 
 export function resolveConfigPaths(config: Config, sourceRoot: string) {
+  
+  if(!sourceRoot) {
+    sourceRoot = '';
+  }
+  
```

Comment:
That would do the trick no?
```suggestion
export function resolveConfigPaths(config: Config, sourceRoot = '') {
```

Reply:
Yep, better ! 
Focused on explain why it did not work well... 
;)